src/Controller/WebserviceController.php

```diff
@@ -170,6 +170,11 @@ public function webonyxAction(Service $service, LocaleServiceInterface $localeSe
             ];
         }
 
+        header('Access-Control-Allow-Origin: ' . $_SERVER['HTTP_ORIGIN']);
```

Comment:
Are you sure `HTTP_ORIGIN` is always available? See 
https://stackoverflow.com/questions/41231116/serverhttp-origin-doesnt-work
Consider maybe use `Access-Control-Allow-Origin: *`?

Reply:
Thanks for your input, I agree it would make sense to have a fallback. The reason why I've decided for HTTP_ORIGIN over * is because * won't work when credentials are sent. For example, on our staging platform, we're securing it with basic auth. To avoid entering the credentials twice, we're using the credentials forwarding option. But this won't work with *.

> For requests without credentials, the literal value "*" can be specified, as a wildcard; the value tells browsers to allow requesting code from any origin to access the resource. Attempting to use the wildcard with credentials will result in an error.

See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin

For the case when it's undefined, we can fall back to *, what do you think?